config/user-commands.m4

```diff
@@ -96,12 +99,20 @@ dnl # These commands may take different command line arguments.
 dnl #
 AC_DEFUN([ZFS_AC_CONFIG_USER_COMMANDS_LINUX], [
 	AC_PATH_TOOL(BLOCKDEV, blockdev, "")
+	AC_PATH_TOOL(CHACL, chacl, "")
```

Comment:
Similarly add a `Requires: acl` to pull in the acl package which provides chacl, getfacl, setfacl.